lang/en.json

```diff
@@ -336,6 +336,8 @@
   "YZECORIOLIS.MigratedDP": "Darkness Points System Migrated.",
   "YZECORIOLIS.HPBonus": "HP Bonus",
   "YZECORIOLIS.MPBonus": "MP Bonus",
+  "YZECORIOLIS.HPInjury": "Perm. HP Reduction",
```

Comment:
Just an idea. Maybe we don't need the `Perm` in the text. I think it can be assumed it is applicable as long as you have the injury on your sheet (like with the talent text just above).

If we go this way, we might want to change the key from `YZECORIOLIS.HPInjury` to `YZECORIOLIS.HPReduction` for consistency.

Reply:
Or maybe `HP Penalty`?

Reply:
Penalty sounds wrong in my head but a consistent naming should be rational.
I guess I will change it together with your reply on the template.